lib/src/draw_config_context.dart

```diff
@@ -167,6 +167,7 @@ class DRAWConfigContext {
     String authorizeUrl,
     String siteName = 'default',
   }) {
+    return;
```

Comment:
Was there something not working with the Config object?

Reply:
Ah, yes, something was broken, can't remember what.